src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -6382,27 +6382,28 @@ bool Unit::HandleDummyAuraProc(Unit* victim, uint32 damage, AuraEffect* triggere
                 }
                 else
                 {    // Check Party/Raid Group
-                    if (Group* group = ToPlayer()->GetGroup())
```

Comment:
I tested it and I was using this fix for a long time and none found any issue related to it.